.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: dbt Labs Community
+    url: https://getdbt.slack.com/archives/CBSQTAPLG
```

Comment:
Do we want to point them towards the signup page? https://www.getdbt.com/community/join-the-community